viahtml/context.py

```diff
@@ -156,6 +156,10 @@ def make_absolute(self, url, proxy=True):
         :param url: URL to convert
         :param proxy: Make the path relative with regards to Via rather than
             the original site
+        :param rewrite_fragments: Enable making fragment only URLs absolute
```

Comment:
It looks like `make_absolute` is currently called in two places:

1. To handle the `Location` field in a redirect
2. To rewrite anchor links in the page.

This made me wonder - does a fragment-only URL even make sense in the `Location` header? Some testing found that it leads to an infinite/broken redirect error in browsers - since the fragment is not sent to the server, it just becomes a redirect-to-self.

So the `rewrite_fragments` param is not strictly needed at present, but it might be needed if `make_absolute` is used elsewhere in future, and setting it to `True` seems a safe default.